olive/passes/onnx/gptq.py

```diff
@@ -0,0 +1,254 @@
+            ),
+            "static_groups": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Use static groups for quantization. Default value is False.",
+            ),
+            "true_sequential": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Use true sequential for quantization. Default value is False.",
+            ),
+            "desc_act": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Use descriptive activation for quantization. Default value is False.",
+            ),
+            "sym": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Symmetric quantization. Default value is False.",
+            ),
+            "data_config": PassConfigParam(
+                type_=Union[DataConfig, Dict],
+                default_value=None,
+                description="""
+                    Data config for quantization. Default value is None.
+                """,
+            ),
+            # TODO(trajep): consider to use data_config to implement the functionality of dataloader_func.
+            "dataloader_func": PassConfigParam(
+                type_=Union[Callable, str],
+                default_value=None,
+                description="""Function/function name to generate dataset for quantization.
+                The returned datasets is a list of tokenized data
+                (e.g. [{ 'input_ids': [ 1, 100, 15, ... ],'attention_mask': [ 1, 1, 1, ... ]},...]).
+                Default is None.
+                """,
+            ),
+            "dataloader_func_kwargs": PassConfigParam(
+                type_=Dict[str, Any],
+                default_value=None,
+                description="Keyword arguments for dataloader_func. Default value is None.",
+            ),
+            "export_optimum": PassConfigParam(
+                type_=bool,
+                default_value=True,
+                description="Whether to use optimum to export ONNX model. Default value is True.",
+            ),
+            "extra_args": PassConfigParam(
+                type_=dict,
+                default_value=None,
+                description="Extra arguments to pass to the `optimum.exporters.onnx.onnx_export_from_model` function.",
+            ),
+            "use_dynamo_exporter": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Whether to use dynamo_export API to export ONNX model when export_optimum=False.",
+            ),
+            "target_opset": PassConfigParam(
+                type_=int,
+                default_value=13,
+                description="The version of the default (ai.onnx) opset to target when export_optimum=False.",
```

Comment:
do we need them?

Reply:
It is needed in config because the export func in conversion extract it from config https://github.com/microsoft/Olive/blob/9d780ea9c356c4fe0bf4eb8971c92f412b1f47fe/olive/passes/onnx/conversion.py#L262-L271